### PR TITLE
Add authorization with devise 

### PR DESCRIPTION
**Pull request by JonahKayizzi**
This pull request consists of the following changes:

- Removed current_user method in ApplicationController in order to use logged in user from devise
- Installed and setup devise and implemented the following
`Can register a new user.`
`User logs in with a combination of email and password.`
`Hashed password should be stored in the database.`
`Ask for confirmation of email.`
`Can reset password.`
- Modify the views in devise and added styling.

### CODE REVIEW

File: app/views/devise/passwords/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<h2>Forgot your password?</h2>
```
Comment (rustam-tolipov) [inline app/views/devise/passwords/new.html.erb:1]: **You did great job, your confirmabe method is working perfect.** 

- [ ] However, I'm facing an issue with recoverable with your devise. If I enter my email I'm facing with this error. Kindly watch the video. Thank you!


https://user-images.githubusercontent.com/69011963/216399038-1cbf68ad-33de-4c13-813b-9e6dc8a2fe57.mov

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hello @JonahKayizzi,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Check the comments under the review._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/devise/passwords/new.html.erb (hunk 1 above)
Comment (JonahKayizzi) [reply app/views/devise/passwords/new.html.erb:1]: Thanks for the feedback. This bug has been fixed in my next commit

**Review by Davidosky007 — APPROVED** [review]
Hi @JonahKayizzi 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* No linter errors 👍 
* Good looking readme file 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**